src/edge-runtime.d.ts

```diff
@@ -19,6 +21,11 @@ declare namespace Supabase {
      * Automatically abort the request to the model after specified time (in seconds). Applies only for LLMs like `mistral` (default: 60)
      */
     timeout?: number
+
+    /**
+     * Mode for the inference API host. (default: 'ollama')
+     */
+    mode?: 'ollama' | 'openaicompatible'
   }
```

Comment:
```suggestion
     */
    mode?: 'ollama' | 'openaicompatible',
    signal?: AbortSignal
  }
```

https://github.com/supabase/edge-runtime/pull/383/files#diff-d163a7d8dc5a27b29104baa126607236b805e35ff3eb15cb294a96f64a3cf58bR126-R127

Reply:
cc @thorwebdev

Reply:
This is used when a user wants to arbitrarily abort a request or streaming response from the inference API in the middle.